apps/08jquery-towers-of-hanoi/script.js

```diff
@@ -1,5 +1,35 @@
 'use strict';
 
+var $block = null;
 $(document).ready(function() {
-  // Put app logic here
+  $('[data-stack]').on('click', function() {
+    //This is were the blocks will be moved from one stack to another stack
+    if ($block === null) {
+
+      $block = $(this).children().last().detach();
+
+    } else {
+      if (
+        //Moving the blocks to either stack
+        $($block).data('block') < $(this).children().last().data('block') ||
+        $(this).children().length === 0) {
+        $(this).append($block);
+        $block = null;
+
+      }
+    }
+    //This is running after every move to make sure that it's a legal move
```

Comment:
This comment's not really accurate. We're not checking for move legality, that happens on line 14